shotover-proxy/tests/test-configs/kafka/cluster-3-racks/topology-rack1.yaml

```diff
@@ -6,13 +6,16 @@ sources:
       chain:
         - KafkaSinkCluster:
             shotover_nodes:
-              - address: "localhost:9191"
+              - address_for_peers: "localhost:9191"
```

Comment:
should we specifically use different addresses for `address_for_peers` in this test case to ensure `check_peers` task can use different addressed?

Reply:
good idea, I did it for cluster-2-racks/topology-rack*.yaml